src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+
+import { LoaderSize } from "@type/index";
+import classNames from "classnames";
+
+import styles from "./Loader.module.scss";
+
+export type LoaderProps = {
+  loading?: boolean;
+  size?: LoaderSize;
+  className?: string;
+  color?: string;
+};
+
+const Loader: FC<LoaderProps> = ({
+  loading = true,
+  size = LoaderSize.m,
+  className,
+  color,
+}: LoaderProps) => {
+  if (!loading) return null;
+  let loader = classNames(styles.testClass);
+  if (className) loader += className;
+  if (size === "s") {
+    loader = classNames(styles.testClass, styles.small);
+  } else if (size === "l") {
+    loader = classNames(styles.testClass, styles.large);
+  }
```

Comment:
всё это можно сделать в три строки
```suggestion
  const classes(styles.testClass, className, {
    [styles.small]: size === 's',
    [styles.large]: size === 'l',
  });
```